src/Fixtures/index.js

```diff
@@ -276,10 +1112,10 @@ export default {
       ],
     },
     {
-      id: 13,
+      id: 51,
       type: 'fitness', // vs technical
-      source: 'Rise Up',
-      title: 'Hot Box',
+      source: '',
+      title: 'Yoga etirement',
```

Comment:
Yoga Stretching, no ? ;)

Reply:
yes

Reply:
For the moment the Index.js doesn't have a good structure. I will complete with real data later